app/controllers/products_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ProductsController < ApplicationController
+  def index
+  end
+
+  def about_us
+  end
```

Comment:
😳Why is this method defined?

Reply:
To navigate to another page based on our design sprint. After Oliver's recommendation the action is removed.